editor/js/ui/editor.js

```diff
@@ -782,7 +787,12 @@ RED.editor = (function() {
             var clear = $('<button class="editor-button editor-button-small"><i class="fa fa-times"></i></button>').appendTo(iconForm);
             clear.click(function(evt) {
                 evt.preventDefault();
-                var iconPath = RED.utils.getDefaultNodeIcon(node._def, node);
+                var iconPath;
+                if (node.type === "subflow") {
```

Comment:
This logic should be inside `RED.utils.getDefaultNodeIcon`

Reply:
Thank you very much for checking this and another one.  I will fix both of them and will push the commit soon.